specification/informatica/Informatica.DataManagement/main.tsp

```diff
@@ -0,0 +1,736 @@
+import "@typespec/rest";
+import "@typespec/openapi";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-apiview";
+import "@azure-tools/typespec-azure-resource-manager";
+
+@armProviderNamespace
+@service({
+  title: "Informatica.DataManagement",
+  description: "Azure Native Services Informatica DataManagement",
+})
+@versioned(Versions)
+@armCommonTypesVersion("v3")
```

Comment:
Why v3? since it is new api version please consider moving to latest v5 version, you have Lint warning for same in the PR.

Reply:
Used v5 to keep the the specs identical to https://github.com/Azure/azure-rest-api-specs-pr/pull/16995 , which is out latest preview version in the rest-api-specs private repo. Any versions we release after this version in this repo will use v5 as armCommonTypesVersion.